assets/js/googlesitekit/datastore/site/urls.test.js

```diff
@@ -77,6 +77,163 @@ describe( 'core/site urls', () => {
 			} );
 		} );
 
+		describe( 'getDocumentationLinkURL', () => {
```

Comment:
Once you validate with the `invariant` method, could you add a test case to verify the required param? Please check other datastores for reference.